src/connectors/loopio/functions/create_project.ts

```diff
@@ -34,7 +34,13 @@ export default DefineConnector({
         enum: ["RFP", "RFI", "DDQ", "SQ", "PP", "OTHER"],
       },
     },
-    required: ["project_name", "company_name", "due_date", "project_type"],
+    required: [
+      "project_name",
+      "company_name",
+      "owner",
```

Comment:
This is a new required property, but without it, this connector would fail. So, technically a breaking change, but practically a fixing change.

Reply:
Sounds good 👍